packages/schemas/tables/saml_application_sessions.sql

```diff
@@ -0,0 +1,32 @@
+/* init_order = 2 */
+
+create table saml_application_sessions (
+  tenant_id varchar(21) not null 
+    references tenants (id) on update cascade on delete cascade,
+  /** The globally unique identifier of the session. */
+  id varchar(32) not null,
+  application_id varchar(21) not null
+    references applications (id) on update cascade on delete cascade,
+  /** The identifier of the SAML SSO auth request ID, SAML request ID is pretty long. */
+  saml_request_id varchar(128),
+  /** The identifier of the OIDC auth request state. */
+  oidc_state varchar(32),
+  /** When checking the OIDC auth state, we should have this flag to prevent replay attack. */
+  is_oidc_state_checked boolean not null default false,
```

Comment:
When do we need this? How about deleting the oidc_state immediately after it is being consumed?